docs/DEVELOPER.md

```diff
@@ -0,0 +1,61 @@
+Oengus developer environment
+============================
+
+This instruction provides steps on how to set up a development environment to work on Oengus backend in Ubuntu or other
+Debian-based GNU/Linux distributions. See [DOCKER.md](./DOCKER.md) for Docker instructions.
+
+Prerequisites
+-------------
+
+Install packages which are needed for running Oengus. JDK 16 or later is required. Just install the latest JDK available
+in your distribution. For example, for Java 17:
+
+    $ sudo apt install openjdk-17-jdk postgresql
+
+Source code
+-----------
+
+    $ git clone git@github.com:esamarathon/oengusio.git
+    $ cd oengusio
+
```

Comment:
Most people will use git over http
```suggestion
```sh
git clone https://github.com/esamarathon/oengusio.git
cd oengusio
\```
```

Reply:
Oops, misclicked this comment.

> Most people will use git over http

Oof. I'm sorry for them. Using Git over SSH is better and I would rather encourage its usage.